### ISSUE TITLE
Smart Guides UI draw over HTML Design Space UI

### ISSUE THREAD

**Issue by chadrolfs** [adobe-photoshop/spaces-design#1632]
It should be limited to canvas area

**Comment by chadrolfs** [comment]
![smartg-over-html-layer](https://cloud.githubusercontent.com/assets/8442631/8285655/9b9e3c80-18b9-11e5-878c-7e3544935629.png)

**Comment by chadrolfs** [comment]
related to #1019

**Comment by iwehrman** [comment]
I reopened #1019 for this, but maybe we should leave that one closed and keep this?

**Comment by volfied** [comment]
This is related to #1019, but it definitely is not the same issue. Let's keep this one open because this is contained in Design Space world.

I personally don't see this as a bug, the smart guides should have priority in the Z-order.

**Comment by chadrolfs** [comment]
https://watsonexp.corp.adobe.com/#bug=4010794

**Comment by iwehrman** [comment]
Am I the only one who thinks this is a bug? @chadrolfs @joelrbrandt @placegraphichere?

**Comment by placegraphichere** [comment]
yes, this is a bug. There are points where those smart guide measurements will continue to appear—even after switching to another app. This is partly something for core I would imagine, since those things seem to be too sticky when used in standard as well. A simple example is using the pan tool when the smart guides are up, they seem to stay in place and not disappear as would be expected. Although we should see if we can simply fix the z order for us first

**Comment by chadrolfs** [comment]
@placegraphichere what you are describing has been fixed (smart guide UI remaining on screen after switching to another app).  This bug, as I understand it, is simply smart guide UI sitting on top of the HTML layer (sometimes drawing over panel UI)

**Comment by iwehrman** [comment]
CC @zoranagee. Maybe we should bump the priority of this?

**Comment by zoranagee** [comment]
People use CMD+ all the time to zoom in and out. And to have your smart guides popping up all over the screen is messy. We should definitely fix this before we ship (we can work around it for MAX).

**Comment by chadrolfs** [comment]
Watson s1p1, assigned to JP.

**Comment by chadrolfs** [comment]
Partially fixed but still can occur in some instances.  Good enough for M3

**Comment by ktaki** [comment]
@chadrolfs: Is this still an issue? It seems CNR for me. Can you try it?

**Comment by chadrolfs** [comment]
Yes, I can still repro but only when moving a layer with Cmd key down.  Very edge case.

![smartguides-overui](https://cloud.githubusercontent.com/assets/8442631/13154368/7aa301dc-d62d-11e5-9002-85dba2f9438c.gif)
